docs/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Filecoin Virtual Machine"
+description: "The Filecoin Virtual Machine (FVM) allows users to write their own smart-contracts and run them against the Filecoin network. This website contains all the documetation for the FVM project, including examples and reference material to help developers build on the FVM."
+lead: "The Filecoin Virtual Machine (FVM) is an interface that developers can use to deploy smart-contracts on the Filecoin network. FVM contracts will introduce the ability to perfom on-chain computation, or the computation of a state. This is the perfect pairing of blockchain storage and computation that Web3 has been waiting for."
+---
+
+# The Filecoin Virtual Machine
+
+The Filecoin Virtual Machine (FVM) is an interface that enables developers to deploy smart contracts on the Filecoin blockchain. FVM support for smart contracts provides the ability to perform on-chain state computations that guide storage deals from pre-commit through committed states. This is the perfect pairing of blockchain storage and computation that Web3 has been waiting for.
```

Comment:
> FVM support for smart contracts provides the ability to perform on-chain state computations that guide storage deals from pre-commit through committed states.

This sentence doesn't mean anything to new readers. Try simplifying things a bit, or just remove this sentence all together.

This whole paragraph is mostly a duplicate of the `lead` text. Let's remove this whole paragraph.

Reply:
Two questions:
1. I had asked before: Why, all of the sudden, are we mixing up metadata with customer-facing data? We've never done this before, that I have seen. The preview I saw displayed the metadata as well as the text intended for the customer. If I have a change for the title and the Lead, should I edit it in the metadata?
2. I don't think c-suite audiences will understand "on-chain computation, or the computation of a state." Can we expect them to know about filecoin and storage provider deals? This needs some sort of explanation....without going into great detail.

Reply:
We discussed this on fvm with raulk and stebalien at (fvm channel)[https://filecoinproject.slack.com/archives/C029MT4PQB1/p1655496096856579], so see how you like the changes.

Reply:
@johnnymatthews Here is Raul's last suggestion:
"The FVM unlocks the ability to deploy new features to the Filecoin blockchain in the form of actors. Actors can access system functionality and can interact with one another. Users can invoke these actors and obtain proof of execution."

This is my latest submission:
"The Filecoin Virtual Machine (FVM) enables programmers to create and deploy smart contracts on the Filecoin blockchain. With smart contracts, programmers can create new features, opening up a wide range of storage and data possibilities, that are provable and traceable."

I thought the introduction of the concept of actors was too technical (more about how it's done, than what it accomplishes). I introduce the concept of actors a little later in the page under Vision and Goals, where I explain the goal to provide speedier processing because actors take up so much space. In that section, they really need to understand what actors are. I used Raul's point that we should address the outcomes the c-suite would look for and appreciate, including "new features." In the spirit of his point, I also added "storage and data" possibilities. I include "proof of execution" (proof of execution of what?) in concept, but take it out of the passive voice which is hard to understand, and instead say "storage and data that are provable and traceable" making it more active.